website/content/docs/auth/kubernetes.mdx

```diff
@@ -189,6 +189,11 @@ token and CA certificate is read once and stored in Vault storage.
 When the service account token expires or is revoked, Vault will no longer be
 able to use the `TokenReview` API and client authentication will fail.
 
+#### Use the system's trust store for CA certificates
+
+Omitting `kubernetes_ca_cert` and setting `disable_local_ca_jwt` to `true` will lead to Vault
+using the system's trust store for TLS certificate verification.
```

Comment:
```suggestion
<Note title="You can use the trust store for CA certificates">

  If you leave `kubernetes_ca_cert` unset and set `disable_local_ca_jwt` to
  `true`, Vault uses the Kubernetes trust store for TLS certificate
  verification.

</Note>
```

Having a separate header doesn't really help with reader understanding. It seemed like this was mostly an FYI style note, so I used that formatting, but if that's not quite right, feel free to swap it for a Tip or Warning instead.

Reply:
Thank you. Will apply most of your suggestion locally.